app/src/main/java/com/elmeradrianv/shesafe/adapters/ReportCardAdapter.java

```diff
@@ -0,0 +1,100 @@
+package com.elmeradrianv.shesafe.adapters;
+
+import android.content.Context;
+import android.util.Log;
+import android.view.LayoutInflater;
+import android.view.View;
+import android.view.ViewGroup;
+import android.widget.TextView;
+
+import androidx.annotation.NonNull;
+import androidx.recyclerview.widget.RecyclerView;
+
+import com.elmeradrianv.shesafe.R;
+import com.elmeradrianv.shesafe.database.Report;
+import com.elmeradrianv.shesafe.fragments.TableViewFragment;
+import com.parse.ParseException;
+import com.parse.ParseQuery;
+
+import java.util.ArrayList;
+import java.util.List;
+
+public class ReportCardAdapter extends RecyclerView.Adapter<ReportCardAdapter.ViewHolder> {
+    public static final String TAG = "PostAdapter";
+    List<Report> reports;
+    Context context;
```

Comment:
I think you can remove this member variable. When you need the context on line 45 you should be able to call something like "parent.getContext()" to get the context from the viewGroup.

In general storing the "context" is bad as it's life cycle can be tricky if you don't let Android manage it for you.